hs_core/views/__init__.py

```diff
@@ -693,6 +771,23 @@ def create_resource_new_workflow(request, *args, **kwargs):
             files=request.FILES.getlist('files'),
             content=res_title
     )
+    # register new resource and set up privileges in HSAccess Database for hydroshare access control
+    try:
+        ha_obj = HSAlib.HSAccess(str(request.user.username), 'unused', django_settings.HS_ACCESS_DB, django_settings.HS_ACCESS_USERNAME,
+                                 django_settings.HS_ACCESS_PASSWORD, django_settings.HS_ACCESS_HOST, django_settings.HS_ACCESS_PORT)
+    except:
+        # unable to connect to the database
+        raise DatabaseError("unable to connect to the database HSAccess.")
+
+    try:
+        homedir = django_settings.IRODS_CWD+"/"+resource.short_id
+        ha_obj.assert_resource(resource_path=homedir, resource_title=res_title, resource_uuid = resource.short_id)
```

Comment:
parameter formatting is inconsistent. last keyword arg has spaces around the `=` the others do not.